src/Logikfabrik.Umbraco.Jet/Logikfabrik.Umbraco.Jet.nuspec

```diff
@@ -15,7 +15,7 @@
     <copyright>Copyright (c) 2016</copyright>
     <tags>Umbraco, Jet, MVC, Code First, uJet</tags>
     <dependencies>
-      <dependency id="UmbracoCms.Core" version="[7.5.6,8.0]" />
+      <dependency id="UmbracoCms.Core" version="[7.6.0,8.0]" />
```

Comment:
Hey thanks for looking at my PR! Yeah, I wasn't too sure about this one. Latest version is now 7.6.3 but these changes are actually for the changes done in Umbraco 7.6.0 (the issue number I linked above). I thought projects still on 7.6.0 should still be able to use this. 

What do you reckon? Should I downgrade the referenced assembly to 7.6.0?